spec/models/miq_expression_spec.rb

```diff
@@ -2094,4 +2094,16 @@
       expect(obj.evaluate(@data_hash)).to eq(false)
     end
   end
+
+  describe ".evaluate_atoms" do
+    it "adds mapping 'result'=>false to expression if expression evaluates to false on supplied object" do
+      expression = {">=" => {"field" => "Vm-num_cpu",
+                            "value" => "2"}}
+      result = described_class.evaluate_atoms(expression, FactoryGirl.create(:vm))
```

Comment:
Minor, but will `build` suffice here? It won't touch the database, making this test faster